resources/assets/components/WithReviewing/index.js

```diff
@@ -61,6 +62,7 @@ const reviewComponent = (Component, data) => {
       this.setState({
         campaign: data.campaign,
         posts: posts,
+        postIds: map(orderBy(posts, 'created_at', 'desc'), 'id'),
```

Comment:
Now that we're using the JSON response for this (rather than the keyed object), do we still need the `orderBy`?

Reply:
OMG YOU ARE RIGHT WE DO NOT 🔥 !!!!